.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (Elixir ${{matrix.elixir}} | Erlang/OTP ${{matrix.otp}})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - elixir: 1.7.4
+            otp: 19.3.6.13
+          - elixir: 1.8.2
+            otp: 20.3.8.26
+          - elixir: 1.9.4
+            otp: 20.3.8.26
+            warnings_as_errors: true # not 1.10 as its --warnigs-as-errors has bugs https://github.com/elixir-lang/elixir/issues/10073
```

Comment:
Why change it? the code compiles fine on 1.10

Reply:
The `--warnigs-as-errors` implementation of Elixir 1.10 [has bugs](https://github.com/elixir-lang/elixir/issues/10073).

Reply:
Do you prefer the scanning stderr workaround proposed [here](https://github.com/elixir-lang/elixir/issues/10073#issuecomment-683347040)?

Reply:
Elixir 1.11 is out so this workaround is no longer necessary

Reply:
Solved in https://github.com/elixir-lsp/elixir_sense/pull/106/commits/a572a61267c82ef99fc554be1b59d33b54dccc81